### PR TITLE
add paragraph in index.html

### PR DESCRIPTION
**Pull request by Igor-11**
Add paragraph in index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,6 @@
 <body>
     <h1>Heading</h1>
     <p>Hello World!</p>
+    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Officiis.</p>
```
Comment (AndriiRafalskyi) [inline index.html:12]: Make first word bold with <strong> tag

File: index.html [hunk 2]
```diff
@@ -9,5 +9,6 @@
 <body>
     <h1>Heading</h1>
```
Comment (AndriiRafalskyi) [inline index.html:10]: Change h1 to h2

**Review by AndriiRafalskyi — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Igor-11) [reply index.html]: Change first word on bold

File: index.html (hunk 2 above)
Comment (Igor-11) [reply index.html]: Changed h1 to h2

**Review by AndriiRafalskyi — APPROVED** [review]

**Pull request merged**